src/test/ts/browser/PluginTest.ts

```diff
@@ -1,17 +1,202 @@
+      '</ul>'
+  })
+
+  listStyleTest('Lower alpha style', async (editor) => {
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'lower-alpha')
+  }, {
+    inputContent: '',
+    type: 'ol',
+    style: 'lower-alpha',
+    expected: '<ol style="list-style-type: lower-alpha;">\n' +
+      '<li>&nbsp;</li>\n' +
+      '</ol>'
+  })
+
+  listStyleTest('Lower greek style', async (editor) => {
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'lower-greek')
+  }, {
+    inputContent: '',
+    type: 'ol',
+    style: 'lower-greek',
+    expected: '<ol style="list-style-type: lower-greek;">\n' +
+      '<li>&nbsp;</li>\n' +
+      '</ol>'
+  })
+
+  listStyleTest('Lower roman style', async (editor) => {
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'lower-roman')
+  }, {
+    inputContent: '',
+    type: 'ol',
+    style: 'lower-roman',
+    expected: '<ol style="list-style-type: lower-roman;">\n' +
+      '<li>&nbsp;</li>\n' +
+      '</ol>'
+  })
+
+  listStyleTest('Upper alpha style', async (editor) => {
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'upper-alpha')
+  }, {
+    inputContent: '',
+    type: 'ol',
+    style: 'upper-alpha',
+    expected: '<ol style="list-style-type: upper-alpha;">\n' +
+      '<li>&nbsp;</li>\n' +
+      '</ol>'
+  })
+
+  listStyleTest('Upper roman style', async (editor) => {
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'upper-roman')
+  }, {
+    inputContent: '',
+    type: 'ol',
+    style: 'upper-roman',
+    expected: '<ol style="list-style-type: upper-roman;">\n' +
+      '<li>&nbsp;</li>\n' +
+      '</ol>'
+  })
+
+  listStyleTest('Setting LI items with Padding Value', async (editor) => {
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'upper-roman')
+    await TestUtils.pSetInputFieldValue(editor, 'Padding Left', '15')
+  }, {
+    inputContent: '',
+    type: 'ol',
+    style: 'upper-roman',
+    expected: '<ol style="list-style-type: upper-roman;">\n' +
+      '<li style="padding-inline-start: 15px;">&nbsp;</li>\n' +
+      '</ol>'
+  })
+
+  it('Applying styles to the selected list only', async () => {
+    const editor = hook.editor();
+    editor.setContent('<ul><li>test</li><ul><li>test2</li></ul></ul>');
+    TinySelections.setSelection(editor, [0, 1], 0, [0, 1], 1);
+    TinyUiActions.clickOnToolbar(editor, 'button[aria-label="shan-enhanced-list"]');
+    await TinyUiActions.pWaitForDialog(editor);
+    await TestUtils.pSetListBoxItem(editor, 'List Style', 'upper-roman')
+    await TestUtils.pSetInputFieldValue(editor, 'Padding Left', '15')
+    await TestUtils.pSetListBoxItem(editor, 'Styles to Applied to', 'The selected list')
+    TinyUiActions.submitDialog(editor);
+    TinyAssertions.assertContent(editor, '<ul>\n' +
+      '<li>test</li>\n' +
+      '<ul style="list-style-type: upper-roman;">\n' +
+      '<li style="padding-inline-start: 15px;">test2</li>\n' +
+      '</ul>\n' +
+      '</ul>');
+  });
```

Comment:
This looks like it could be refactored as well, like `listStyleTest`

Reply:
👍  Refactored to include section in the `Definition` interface and able to reuse the `listStyleTest`